docs/guide/plugins/chai-webdriverio.md

```diff
@@ -0,0 +1,72 @@
+name: chai-webdriverio
+category: plugins
+tags: guide
+index: 3
+title: Chai-WebdriverIO
+---
+# chai-webdriverio ![Travis badge](https://travis-ci.org/marcodejongh/chai-webdriverio.svg?branch=master) [![Greenkeeper badge](https://badges.greenkeeper.io/marcodejongh/chai-webdriverio.svg)](https://greenkeeper.io/)
```

Comment:
I wasn't sure if I should remove the badge's or not, so let me know if I should remove them